apps/mobile/mobile-app/app/screens/ItemsDetail.tsx

```
@@ -0,0 +1,99 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, ScrollView } from 'react-native';
+
+const ItemDetails = ({ route }: { route: any }) => {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)